.github/golangci.yaml

```diff
@@ -51,7 +51,6 @@ linters:
     - tagliatelle # Checks struct tags
     - tenv # Detects using os.Setenv instead of t.Setenv
     - testableexamples # Checks if examples are testable (have expected output)
-    - unparam # Finds unused params
```

Comment:
Why remove 😢

Reply:
I would like to keep some key decoding functions that makes sense for future data encoded at key level:

```
golangci-lint run --config .github/golangci.yaml
storage/encode.go:45:39: decodeUint32Ascending - result 0 ([]byte) is never used (unparam)
func decodeUint32Ascending(b []byte) ([]byte, uint32, error) {
                                      ^
storage/encode.go:159:56: decodeUnsafeStringAscending - result 1 (string) is never used (unparam)
func decodeUnsafeStringAscending(b, r []byte) ([]byte, string, error) {
```

Do you know how to mark these functions so they are ignored by the linter?